ocaml/libs/http-lib/http_client.ml

```diff
@@ -119,6 +119,8 @@ let response_of_fd_exn_slow fd =
       ; additional_headers= !headers
       ; body= None
       }
+  | [] ->
+      raise End_of_file
```

Comment:
This could also be `raise_notrace` if we only use this for control-flow and don't want to log a stacktrace here.